apps/cookbook/src/app/shared/showcase-properties/showcase-property.ts

```diff
@@ -6,9 +6,21 @@ export interface ShowcaseProperty {
   preserveInputValuesWhitespaces?: boolean;
 }
 
-export interface ShowCasePropertyColumns {
+export interface ShowcasePropertyColumns {
```

Comment:
Consider renaming interface as it's used for both properties, methods and events - alternatively creating a base interface (i.e. name + description) and 3 explicit interfaces for properties, methods and events